.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/dmlc/dmlc-core
 [submodule "rabit"]
 	path = rabit
-	url = https://github.com/dmlc/rabit
+	url = https://github.com/chenqin/rabit
```

Comment:
can you update this to the official repo again? and run some integration test (with large scale dataset)?